adapters/bmtm/brightmountainmedia_test.go

```diff
@@ -11,12 +11,7 @@ import (
 )
 
 func TestJsonSamples(t *testing.T) {
-	bidder, buildErr := Builder(openrtb_ext.BidderBmtm, config.Adapter{
-		Endpoint: "https://example.com/api/pbs"})
-
-	if buildErr != nil {
-		assert.NoError(t, buildErr, fmt.Sprintf("Builder returned unexpected error: %s", buildErr.Error()))
-	}
-
+	bidder, buildErr := Builder(openrtb_ext.BidderBmtm, config.Adapter{Endpoint: "https://example.com/api/pbs"})
+	assert.NoError(t, buildErr, fmt.Sprintf("Builder returned unexpected error: %s", buildErr.Error()))
```

Comment:
Looks like appending the actual error message here is causing the tests to panic because we expect `buildErr` to be nil and we're trying to pass `buildErr.Error()` to the `assert.NoError` function. Can you please remove that here and in other places to avoid the test to panic? Also, the `assert` library itself will print the actual error message in case `buildErr` ends up being not nil so we don't need to explicitly do that

Reply:
Please use the standard test function:
```
func TestJsonSamples(t *testing.T) {
  bidder, buildErr := Builder(openrtb_ext.Bidder{Bidder}, config.Adapter{
    Endpoint: "http://whatever.url"})
  
  if buildErr != nil {
    t.Fatalf("Builder returned unexpected error %v", buildErr)
  }
  
  adapterstest.RunJSONBidderTest(t, "{bidder}test", bidder)
}
```

Reply:
@SyntaxNode That's how they had it before. I actually suggested them to change it to using the `assert` library because that makes the code a little more concise and that's what we use in the core code for tests. However, I understand that this is how all other adapters have it currently. This was more intended to perhaps make that change moving forward. For now, I have approved the PR since this is not critical. 

Sorry about the different directions @BrightMountainMediaInc